plugins/action/bonsai_asset.py

```diff
@@ -67,6 +67,7 @@ def validate_arguments(args):
 
         validate("name", args, required=True, typ=str)
         validate("version", args, required=True, typ=str)
+        validate("namespace", args, required=False, typ=str)
```

Comment:
This line should be removed since we are not using the `namespace` parameter in our code. As the comment in this function states, the asset module will validate the parameters of this kind (`auth` was the only one until now).